integrations/elasticsearch/src/elasticsearch_haystack/embedding_retriever.py

```diff
@@ -64,17 +64,18 @@ def from_dict(cls, data: Dict[str, Any]) -> "ElasticsearchEmbeddingRetriever":
         return default_from_dict(cls, data)
 
     @component.output_types(documents=List[Document])
-    def run(self, query_embedding: List[float]):
+    def run(self, query_embedding: List[float], top_k:int = None):
         """
         Retrieve documents using a vector similarity metric.
 
         :param query_embedding: Embedding of the query.
+        :param top_k: Maximum number of Documents to return
```

Comment:
Thanks for fixing the docs! It's out of the scope of this PR, but would you mind adding a similar docstring to the `run` method of the other retriever component?

Reply:
@masci 

The changes you suggested will be done soon.

**Question:**  When you mentioned other retriever components you mean in the main haystack right?

Reply:
@sahusiddharth no I mean the bm25 retriever in this integration, see https://github.com/deepset-ai/haystack-core-integrations/blob/main/integrations/elasticsearch/src/elasticsearch_haystack/embedding_retriever.py#L69-L72